docs/source/building-supergraphs/router.mdx

```diff
@@ -3,19 +3,28 @@ title: The router
 description: The entry point to your supergraph
 ---
 
-import FederationArchitecture from '../../shared/diagrams/federation-architecture.mdx';
-
 After you set up at least one federation-ready [subgraph](./subgraphs-overview/), you can configure a **router** (formerly known as a gateway) to sit in front of your subgraphs. The router serves as the entry point to your supergraph, and it executes incoming operations across one or more of your subgraphs:
 
-<FederationArchitecture />
```

Comment:
This was the only remaining use of this shared diagram. I might _re_-factor-out a diagram in the future but for now just replacing in-line